apps/src/lib/levelbuilder/lesson-editor/LessonEditor.jsx

```diff
@@ -206,6 +208,14 @@ export default class LessonEditor extends Component {
           <ResourcesEditor resources={this.props.resources} />
         </CollapsibleEditorSection>
 
+        <CollapsibleEditorSection
+          title="Objectives"
+          collapsed={true}
+          fullWidth={true}
+        >
+          <ObjectivesEditor objectives={this.props.objectives || []} />
```

Comment:
Can you make this just this.props.objectives and move the setting of empty array part out to the lesson/edit.js file? That is the pattern we have been using for other things so it would be good to keep it consistent